linux/install/install.sh

```diff
@@ -1,8 +1,45 @@
 #!/bin/bash
 
-hash lsb_release 2>/dev/null || { echo >&2 "Unable to detect lsb_release - manual install required.  Aborting."; exit 1; }
+APP_PATH=`echo $0 | awk '{split($0,patharr,"/"); idx=1; while(patharr[idx+1] != "") { if (patharr[idx] != "/") {printf("%s/", patharr[idx]); idx++ }} }'`
+APP_PATH=`cd "$APP_PATH"; pwd`
 
-ID=`lsb_release -is`
-VERSION=`lsb_release -rs`
+SKM_PATH=`cd "$APP_PATH/../.."; pwd`
+
```

Comment:
Can we add something in here to also install the required packages?

The old code was starting to detect OS version to work out what to install.

```bash
ID=`lsb_release -is`
VERSION=`lsb_release -rs`
```

Reply:
As in have a case statement for the different distro?
Do you have a list of distros you would like to support?
Although I'd recommend using either `/etc/os-release` or `hostnamectl`, as `lsb-release` isn't always installed.
Do distro version number have a big affect package names? (Arch is a rolling release)

Reply:
I've been looking into dependencies and testing for a few distros via docker.
I decided to make a repo, and intend to slowly automate more of the testing.
https://github.com/rodneyp290/skm-linux-install-test-dockerfiles

Currently, I have Dockerfiles working for Arch, Debian, Fedora, Kali, and Ubuntu. 
Although Debian, Kali, Ubuntu have an issue with g++ compiling - says something about recompiling with -fPIC - but clang works fine.
Did you have a minimal set of distros you wanted splashkit to support?

With those ones working, hopefully I'll get around to adding the os detection and dependency installation to `skm linux install` soon

Reply:
Is the `-fPIC` an error or warning? This is something we could look at adding to the compile options if we can get it to work across Linux distributions.

Reply:
Not sure what issues there are between versions, but I do recall there was some issues...

Reply:
I had a quick look over lunch. It was an error. The commit decac92 (below fixed it). 
Although I probably should have double checked all the other distros before posting.
Will confirm later.